### DISCUSSION TITLE
great product!!!

### DISCUSSION THREAD

**Discussion by sunneydev** [coollabsio/coolify#1864] [Improvement Requests]
wow craxzy product guys!!! i ran it today and it automatically restarted my whole docker setup and it broke all of my production stuff to the point where its not recoverable. very cool

**Comment by DamonGuzman** [comment]
@sunneydev Ideally, some information on what broke, how it broke, and ideally logs attached would be helpful. I see this project as an amazing achievement of open source and we should all help each other to make sure it works for everyone. Is there anything specific you can add to help others assist you?

**Reply by sunneydev** [reply]
it is absolutely insane to restart docker processes without asking. surely it will break something for some. it instantly lost my trust for the project and its reliability as well as integrity of the person developing it.

**Reply by peaklabs-dev** [reply]
@sunneydev Did you not use a fresh VPS?

**Reply by sunneydev** [reply]
nope, my production one

**Reply by peaklabs-dev** [reply]
Why would you do that. 
**Updating the docs so nobody else makes this mistake. Make sure to use a fresh VPS...** Sorry that happen. The docs definitely need some work.

**Reply by peaklabs-dev** [reply]
@sunneydev on another note. The install script should not cause issues with a already running containers. Still using a fresh VPS is recommended. If you have more error details it would help in improving coolifys installation?

**Reply by peaklabs-dev** [reply]
@sunneydev and @DamonGuzman the small update in the docs will be added with this PR: https://github.com/coollabsio/documentation-coolify/pull/49

**Comment by peaklabs-dev** [comment]
@sunneydev and @DamonGuzman The new warning in the docs is now available:
<img width="813" alt="image" src="https://github.com/user-attachments/assets/5a6bcc34-0ef5-4122-b051-3645ac7df5e1">
--> If @DamonGuzman has more steps to reproduce and a detailed error code, please open an issue so the install script can be adjusted and fixed: https://github.com/coollabsio/coolify/issues
